examples/qml_features/rust/build.rs

```diff
@@ -9,19 +9,26 @@ use cxx_qt_build::CxxQtBuilder;
 
 fn main() {
     CxxQtBuilder::new()
-        .file("src/containers.rs")
-        .file("src/custom_base_class.rs")
-        .file("src/custom_parent_class.rs")
-        .file("src/invokables.rs")
-        .file("src/multiple_qobjects.rs")
-        .file("src/nested_qobjects.rs")
-        .file("src/serialisation.rs")
-        .file("src/signals.rs")
-        .file("src/singleton.rs")
-        .file("src/properties.rs")
-        .file("src/threading.rs")
-        .file("src/types.rs")
-        .file("src/uncreatable.rs")
+        .qml_module(
```

Comment:
was the plan still to move the arguments into a struct like

```rust
    CxxQtBuilder::new()
        .qml_module(QmlModule {
            uri: "com.example.qobject"
            version_major: 1
            version_minor: 0
            rust_files: ["src/cxxqt_object.rs"]
            qml_files: ["ui/foo.qml"]
        })
        .build();
```

As when we have mutliple args having named args is much nicer. And having a struct allows in the future the potential of builder pattern too.

Reply:
Maybe, but then I think the struct would have to deal with lifetimes. This implementation is easy and flexible in what the user can provide with CxxQtBuilder::qml_module taking `&str` and `&[impl AsRef<Path>]` then converting those to owned values to create the struct.